packages/pluggableWidgets/image-viewer-web/tests/e2e/specs/errors.spec.ts

```diff
@@ -0,0 +1,80 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+import ImageViewer from "../objects/imageViewer.widget";
+
+// eslint-disable-next-line jest/no-disabled-tests
+xdescribe("Image viewer", () => {
```

Comment:
I assume this one is also not needed as we now throw consistency checks

Reply:
True story! Removed!